docs/book/operator-guide/backup-restore-guide.md

```diff
@@ -0,0 +1,144 @@
+# Backup and restore guide
+
+## Overview
+
+Walrus storage nodes provide backup and restore functionality for the primary database containing
+blob data. This guide covers configuration requirements, operational procedures, and best practices
+for both automated and manual backup processes, as well as restore operations.
+
+## Prerequisites
+
+- Storage node must be running with appropriate permissions to create backups
+- Sufficient disk space for backup storage (recommended: separate physical volume)
+- Unix/Linux operating system with support for Unix domain sockets
+- `walrus` user account with appropriate permissions
+
+### Local administration socket configuration
+
+The backup system communicates with running storage nodes through a Unix domain socket. To enable
+this functionality:
+
+1. **Configure the administration socket path** in your node configuration file:
+
+   ```yaml
+   admin_socket_path: /opt/walrus/admin.socket
+   ```
+
+1. **Restart the storage node** to initialize the socket:
+
+   ```bash
+   sudo systemctl restart walrus-node.service
+   ```
+
+1. **Verify socket creation**:
+
+   ```bash
+   ls -la /opt/walrus/admin.socket
+   ```
+
+```admonish warning
+All administrative operations must be executed under the `walrus` user context to ensure proper
+permissions and security.
+```
+
+## Automated periodic backups
+
+Storage nodes support scheduled automatic backups through checkpoint configuration. Add the
+following configs to your node configuration:
+
+```yaml
+checkpoint_config:
+  # Directory where backups will be stored
+  db_checkpoint_dir: /opt/walrus/checkpoints
+
+  # Number of backups to retain (oldest will be deleted)
+  max_db_checkpoints: 2
+
+  # Backup frequency (example: 4-hour interval)
+  db_checkpoint_interval:
+    secs: 14400  # 4 hours in seconds
+    nanos: 0
+
+  # Sync in-memory data to disk before creating a bakcup
+  sync: true
+
+  # Maximum concurrent backup operations
+  max_background_operations: 1
+
+  # Enable/disable automated backups
+  periodic_db_checkpoints: true
+```
+
+## Manual backup creation
+
+Create on-demand backups using the `local-admin` command:
+
+```bash
+# Basic backup command
+sudo -u walrus walrus-node local-admin \
```

Comment:
I imagine this will be along running command and wonder if this command need to be run with `tmux` or `screen` so we keep the session live? If not, maybe worth mentioning it here.

Reply:
When I tried this, the command did *not* return immediately. Is this just a bug with the `local-admin checkpoint create` command?